packages/filebrowser/src/listing.ts

```diff
@@ -1799,7 +1799,10 @@ export namespace DirListing {
       let modTitle = '';
       if (model.last_modified) {
         modText = Time.formatHuman(new Date(model.last_modified));
-        modTitle = Time.format(new Date(model.last_modified), 'lll');
+        modTitle = Time.format(
+          new Date(model.last_modified),
+          'MMM dd, yyyy hh:mm a'
```

Comment:
Does this indicate that this change is backwards incompatible? Or would format `'lll'` work just fine here, and you've just decided to change it?

Reply:
Answered in PR description, so let's carry out the conversation there.

Reply:
`lll` wouldn't work here as it's specific to moment.js. That's the only instance where I have seen the function called with the `lll` argument, but I guess that if other extensions rely on that syntax then it won't work and therefore isn't backwards compatible.

Reply:
A workaround, of course, would be to accept `lll` and convert it to the appropriate format. However there are many other formats like `lll`l accepted by moment.js so this approach is probably not bullet proof.

Reply:
Okay, no problem, let's set this as 2.0, then, since we are trying to really keep semver versioning. We're planning on releasing 2.0 around January.